src/assets/img/homepage/gitter-logo.svg

```diff
@@ -0,0 +1 @@
+<svg width="2500" height="2500" viewBox="0 0 256 256" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid"><defs><linearGradient x1="50%" y1="0%" x2="50%" y2="100%" id="a"><stop stop-color="#FB0766" offset="0%"/><stop stop-color="#C50948" offset="100%"/></linearGradient></defs><path d="M0 0h256v256H0V0z" fill="url(#a)"/><path d="M83.914 62.873h12.525v82.661H83.914V62.873zm76.149 20.039h12.524v62.622h-12.524V82.912zm-50.599 0h12.524v110.466h-12.524V82.912zm25.049 0h12.525v110.466h-12.525V82.912z" fill="#FFF"/></svg>
```

Comment:
How do these SVGs look in the dark mode?

Reply:
it's ok, but now we replaced Discord with Gitter!